mini/recursive.go

```diff
@@ -0,0 +1,24 @@
+package mini
+
+import (
+    "github.com/awalterschulze/gominikanren/micro"
+)
+
+/*
+(defrel (very-recursiveo)
+  (conde
+    ((nevero))
+    ((very-recursiveo))
+    ((alwayso))
+    ((very-recursiveo))
+    ((nevero))))
+*/
+func VeryRecursiveO() micro.Goal {
```

Comment:
Maybe this should move to the test file?

Reply:
good idea